packages/web-components/src/components/cbp-toggle/cbp-toggle.scss

```diff
@@ -85,8 +85,8 @@
   --cbp-toggle-text-color: var(--cbp-color-text-darkest);
   --cbp-toggle-text-color-dark: var(--cbp-color-text-lightest);
 
-  --cbp-slider-width: 3.25rem;
-  --cbp-slider-height: 1.75rem;
+  --cbp-toggle-control-width: 3.25rem;
+  --cbp-toggle-control-height: 1.75rem;
   --cbp-circle-diameter: 1rem;
```

Comment:
these last 3 should still use the component namespace.

Reply:
updated as requested